src/command_modules/azure-cli-eventgrid/azure/cli/command_modules/eventgrid/_help.py

```diff
@@ -79,37 +79,37 @@
     """
 helps['eventgrid event-subscription'] = """
     type: group
-    short-summary: Manage event subscriptions for a subscription or resource group.
+    short-summary: Manage event subscriptions.
     """
 helps['eventgrid event-subscription create'] = """
     type: command
-    short-summary: Create a new event subscription to a subscription or resource group.
+    short-summary: Create a new event subscription for a subscription or resource group.
```

Comment:
One thing I have noticed is that `long-summary` is rarely used by command descriptions. It might be worth coming up with a basic guideline for how long/involved we would like short summaries to be before they need to be expanded out -- this could be translated into a linter rule.

Reply:
@sptramer I'll make a note for us to discuss this in our next sync and we can pencil in some rules for this.